Packages/com.unity.inputsystem/InputSystem/Plugins/OnScreen/OnScreenButton.cs

```diff
@@ -39,6 +51,8 @@ public void OnPointerDown(PointerEventData eventData)
         [SerializeField]
         private string m_ControlPath;
 
+        [NonSerialized] bool m_Pressed;
```

Comment:
I dont think you need NonSerialized if the field is already private. Its the default behaviour.

Reply:
Ended up reworking the change which removed the field altogether.